reference/admin.yaml

```diff
@@ -910,3 +968,92 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/403'
+  /admin/per_diem_rates:
+    get:
+      tags:
+        - Per Diem Rates
+      summary: List per diem rates
+      operationId: per_diem_rates_list
+      parameters:
+        - $ref: '#/components/parameters/created_at'
+        - $ref: '#/components/parameters/updated_at'
+        - $ref: '#/components/parameters/display_name'
```

Comment:
per_diem_rate does not have a view like projects/categories, thus querying can only be done on name

Reply:
will fix - thanks